inspirehep/modules/workflows/tasks/actions.py

```diff
@@ -210,6 +210,17 @@ def is_submission(obj, eng):
 
 
 @with_debug_logging
+def add_holdingpen_entry_id(obj, eng):
+    """Add holdingpen entry id for this record in 'submission_number' """
+    source = obj.data.get('acquisition_source')
+    if source:
+        if source.get('method') == "hepcrawl":
+            source['submission_number'] = obj.id
+            return True
+    return False
```

Comment:
Workflows steps don't need return values: they operate with side effects. You can remove this two lines.